src/Splitio-net-core/Services/Events/Interfaces/IEventSdkApiClient.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Splitio.Services.Events.Interfaces
+{
+    public interface IEventSdkApiClient
+    {
+        void SendBulkEvents(string events);
```

Comment:
Why not make SendBulkEvents have a parameter that is the list of events, and internally SendBulkEvents turns them into the json representation?

Reply:
That would be far better. Do you wan't me to adjust Impressions client too? I made it this way to keep consistency

Reply:
Yes please